internal/wire/testdata/BindInterfaceWithValue/want/program_out.txt

```diff
@@ -0,0 +1 @@
+
```

Comment:
To make this a more useful output to compare, we could use `io.Writer` throughout the test and write a message to stdout.